docs/source/conf.py

```diff
@@ -302,7 +302,7 @@ def package_list_from_file(file):
     mocked_packages = []
     with open(file) as fp:
         for ln in fp.readlines():
-            found = [ln.index(ch) for ch in list(",=<>#") if ch in ln]
+            found = [ln.index(ch) for ch in list(",=<>#[]\n") if ch in ln]
```

Comment:
```suggestion
            found = [ln.index(ch) for ch in list(",=<>#[]") if ch in ln]
```
this is not needed and also Winn I think had different